requirements.txt

```diff
@@ -87,25 +85,24 @@ python-json-logger==2.0.7
 python-magic==0.4.27
 pytz==2023.3
 PyYAML==5.4.1
-redis==4.5.4
-requests==2.28.2
+redis==4.5.5
+requests==2.31.0
```

Comment:
I thought we couldn't upgrade requests due to the alpine container version? Was this code deployed to Dev?